en/docs/reference/mediators/payloadfactory-mediator.md

```diff
@@ -172,34 +144,34 @@ The following root variables are available when you format a FreeMarker payload:
   </tr>
   <tr>
     <th>
-      <code>ctx</code>
+      <code>var</code>
     </th>
     <td>
-      You can use the ctx variable to access properties with the 'default' scope. For example, if you have a property named `customer_id` in the default scope, you can get the property in the FreeMarker template by using `ctx.customer_id`.
+      You can use the var variable to access variables. For example, if you have a variable named `customer_id`, you can get the variable in the FreeMarker template by using `var.customer_id`.
```

Comment:
```suggestion
      You can use the <code>var</code> variable to access variables. For example, if you have a variable named <code>customer_id</code>, you can get the variable in the FreeMarker template by using <code>var.customer_id</code>.
```

Shall we use the `<code></code>` tag to format the param/var names and values instead of `  `` ` and ` ' ' ` inside the table? The backticks don't render properly in the UI. Kindly check the other places as well.

Reply:
Fixed with 07ecd88